nixos/modules/services/misc/plex.nix

```diff
@@ -30,6 +30,30 @@ in
         default = "plex";
         description = "Group under which Plex runs.";
       };
+
+
+      managePlugins = mkOption {
```

Comment:
Plex's plugin directory is stored in `/var/lib/plex`. This is necessary because Plex manages some of its own plugins. Unfortunately, that means that we have to clear out symlinks every time it starts up; thus, the option to disable plugin management altogether.